mem-01-b.xml

```diff
@@ -1505,7 +1505,7 @@ Watch Gowron say this: {YouTube video:url:http://youtu.be/kWP0lgIYMBA}</column>
       <column name="examples_de"></column>
       <column name="examples_fa"></column>
       <column name="examples_sv"></column>
-      <column name="examples_ru">Возможно ты умрешь правильно(с достоинством, с честью). Пожелание достойной смерти</column>
```

Comment:
Okay, but now I'm confused about this change here. Is this supposed to be the Russian definition?

Reply:
@dadap How should this be treated?